babel.config.js

```diff
@@ -39,6 +37,7 @@ module.exports = function (api) {
     case "standalone":
       includeRegeneratorRuntime = true;
       unambiguousSources.push("packages/babel-runtime/regenerator");
+      envOpts.exclude = ["transform-typeof-symbol"]; // exclude slow plugins
```

Comment:
`transform-typeof-symbol` is skipped since node.js 0.12, we only need to exclude this plugin when building `@babel/standalone`.

Reply:
We shouldn't even need to exclude since loose + preset-env should do this automatically actually haha

Reply:
oh here: https://github.com/babel/babel/blob/3a0aa8e829ee8b0af9b01df5f581aad6a3f126c0/packages/babel-preset-env/src/get-option-specific-excludes.js#L3